fastlane/Fastfile

```diff
@@ -78,6 +79,42 @@ lane :release do |options|
   github_release(version: version_number)
 end
 
+desc "Creates snapshot pre-release and publishes with the 'snapshot' tag"
+lane :snapshot_release do
+  stable_version   = current_version_number
```

Comment:
One thing we do in Android + iOS is to bump the version number in main to include it's a snapshot: https://github.com/RevenueCat/purchases-android/blob/178691123e9638616f5eff6f17bbb1cf46f2808e/fastlane/Fastfile#L165-L173

This could have some advantages here... Mainly, I believe with this code we would be creating snapshots of the last stable version, which doesn't make sense since snapshots usually refer to a yet "unreleased" version.

So I think we could potentially bump the version in `.version` to the next minor + `-SNAPSHOT` as we do in android/ios or at least calculate the next minor for the current version, instead of using the last released stable version.

Reply:
not sure about this, in practice you'll have multiple PRs merged between released versions, right?

Thinking here would be having some kind of a reference to the last version and then some suffix indicating that it's a different change ..

Agree with you that it could be confusing, but I think incrementing a minor in each PR merge can make things worse